packages/backend/src/services/sms/sms-sender.ts

```diff
@@ -0,0 +1,53 @@
+import { getEnv } from '@helpers';
+import type { ITwilioOptions } from './twilio-sms-provider';
+import { TwilioService } from './twilio-sms-provider';
+
+export interface ISMSSenderService {
+  send(phone: string, message: string): Promise<object>;
+  getById(id: string): Promise<object>;
+}
+
+class SMSSenderService implements ISMSSenderService {
+  private providerService: ISMSSenderService;
+
+  constructor() {
+    this.providerService = new TwilioService({
```

Comment:
Create file `src/providers/sms/TwilioSMSProvider.ts` and move TwilioService there (of course rename class to TwilioSMSProvider). Create interface ISMSProvider and implement TwilioService from it. Recieve TwilioSMSProvider in constructor. Initialize and pass TwillioSMSProvider in `initServices`